gwctl/pkg/printer/gatewayclasses.go

```diff
@@ -95,11 +94,19 @@ func (gcp *GatewayClassesPrinter) PrintDescribeView(resourceModel *resourcedisco
 			{
 				Name: gatewayClassNode.GatewayClass.GetName(),
 			},
-			{
+		}
+
+		if gatewayClassNode.GatewayClass.Spec.Description != nil {
+			views = append(views, gatewayClassDescribeView{
 				ControllerName: string(gatewayClassNode.GatewayClass.Spec.ControllerName),
 				Description:    *gatewayClassNode.GatewayClass.Spec.Description,
```

Comment:
nit: Is it more neat if we simply convert Description to a `*string` and just directly assign:

```
Description: gatewayClassNode.GatewayClass.Spec.Description
```

Alternatively, if we really don't want a pointer, maybe the following is still slightly neater (since it keeps `ControllerName` out of the conditional)?

```
		views := []gatewayClassDescribeView{
			{
				Name: gatewayClassNode.GatewayClass.GetName(),
			},
			{
				ControllerName: string(gatewayClassNode.GatewayClass.Spec.ControllerName),
			},
		}

		if desc := gatewayClassNode.GatewayClass.Spec.Description; desc != nil {
			views = append(views, gatewayClassDescribeView{Description: *desc})
		}
```

Reply:
I didn't want to change the type but sure, why not.  https://github.com/kubernetes-sigs/gateway-api/pull/2894/commits/9cd09fb188032d506368447b23f36f80b7726592